rust/benches/messages/utils.rs

```diff
@@ -0,0 +1,60 @@
+pub mod update {
+    use xaynet_core::{
+        message::{ToBytes, Update},
+        testutils::multipart,
+    };
+
+    fn make_update(
+        dict_len: usize,
+        mask_len: usize,
+        total_expected_len: usize,
+    ) -> (Update, Vec<u8>) {
+        let update = multipart::update(dict_len, mask_len);
+        // just check that we made our calculation right
+        // message size = dict_len + mask_len + 64*2
+        assert_eq!(update.buffer_length(), total_expected_len);
+        let mut bytes = vec![0; update.buffer_length()];
+        update.to_bytes(&mut bytes);
+        (update, bytes)
+    }
+
+    /// Get an update that corresponds to:
+    ///     - 1 sum participant (1 entry in the seed dict)
+    ///     - a 128 bytes serialized masked model
```

Comment:
this doesn't match the actual impl, should it be
```suggestion
    ///     - a 28 bytes serialized masked model
```
?